include/srf/experimental/modules/sample_modules.hpp

```diff
@@ -0,0 +1,229 @@
+/**
+ * SPDX-FileCopyrightText: Copyright (c) 2021-2022, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
```

Comment:
I don't think docs is the right place, since the sample modules are used in various unit tests; I had originally wanted to place them in tests/..., but then I need to add include paths python testing module.

Maybe we should discuss this more.

Reply:
Based on our discussion, I think we can defer this issue to a larger 'testing' library reorganization in a future release: https://github.com/nv-morpheus/SRF/issues/210